alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/ui/ProviderGlobalUIConfig.java

```diff
@@ -58,4 +74,29 @@ public ProviderKey getProviderKey() {
         return providerKey;
     }
 
+    private Collection<String> validateDuplicateNames(FieldValueModel fieldToValidate, FieldModel fieldModel) {
+        try {
+            List<ConfigurationModel> configurations = configurationAccessor.getConfigurationsByDescriptorType(DescriptorType.PROVIDER);
+            if (!configurations.isEmpty()) {
```

Comment:
I don't think we really need this empty check. If the content is empty, we'll stream through an empty list creating another empty list which we're checking later on.

Reply:
I think this is defensive coding and saves us from running a few lines of code that we may not want.